src/components/TaskList.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { AiFillDelete, AiFillEdit } from "react-icons/ai";
+import { useDispatch, useSelector } from "react-redux";
+import { DeleteTask, Edit, Update } from "../features/todos";
+const TaskList = () => {
+  const toDo = useSelector((state) => state.ToDo.value);
```

Comment:
I believe `todo` is one word, If you can research this and improve the naming of the `todo` variable.